cal-itp/benefits
# Chore: setup Nevada County Connects in `test`

Part of #2434

This PR adds the logo and templates required to set up a new transit agency. These are, for the transit agency

- Nevada County (NevCo) Connects logo (small and large)
- `index` template
- `eligibility index` template

and for the enrollment flows

- `enrollment success` template

Setting up the database in `test` is being tracked in the issue.
## To test

To test logos, `core/index--nevco.html`, and `eligibility/index--nevco.html`, add these fixtures (NevCo transit agency and an older adult flow) to your local fixtures (ideally the repo's `local_fixtures.json` so the primary keys of these fixtures don't conflict with your local ones) and reset your database to load them:
```
{
        "model": "core.transitagency",
        "pk": 2,
        "fields": {
            "active": true,
            "slug": "nevco",
            "short_name": "NevCo (test)",
            "long_name": "Nevada County Connects (test)",
            "info_url": "https://www.nevadacountyca.gov/3456/Transit-Services",
            "phone": "530-477-0103",
            "index_template": "core/index--nevco.html",
            "eligibility_index_template": "eligibility/index--nevco.html",
            "eligibility_api_id": "eligibility-api-id",
            "eligibility_api_private_key": 2,
            "eligibility_api_public_key": 3,
            "eligibility_api_jws_signing_alg": "RS256",
            "transit_processor": 1,
            "transit_processor_audience": "audience",
            "transit_processor_client_id": "client-id",
            "transit_processor_client_secret_name": "client-secret-name",
            "staff_group": null,
            "sso_domain": "",
            "customer_service_group": null
        }
    },
{
        "model": "core.enrollmentflow",
        "pk": 6,
        "fields": {
            "system_name": "senior",
            "display_order": 5,
            "claims_provider": 1,
            "claims_scope": "verify:senior",
            "claims_claim": "senior",
            "claims_scheme_override": "",
            "eligibility_api_url": "",
            "eligibility_api_auth_header": "",
            "eligibility_api_auth_key_secret_name": null,
            "eligibility_api_public_key": null,
            "eligibility_api_jwe_cek_enc": "",
            "eligibility_api_jwe_encryption_alg": "",
            "eligibility_api_jws_signing_alg": "",
            "selection_label_template": "eligibility/includes/selection-label--senior.html",
            "eligibility_start_template": "eligibility/start--senior.html",
            "eligibility_form_class": "",
            "eligibility_unverified_template": "eligibility/unverified.html",
            "help_template": "",
            "label": "Older Adult",
            "group_id": "group123",
            "supports_expiration": false,
            "expiration_days": null,
            "expiration_reenrollment_days": null,
            "enrollment_index_template": "enrollment/index.html",
            "reenrollment_error_template": "",
            "enrollment_success_template": "enrollment/success.html",
            "supported_enrollment_methods": "digital,in_person",
            "transit_agency": 2
        }
    }
```
